lexicons/app/bsky/label/defs.json

```diff
@@ -0,0 +1,103 @@
+{
+  "lexicon": 1,
+  "id": "app.bsky.label.defs",
+  "defs": {
+    "labelerView": {
+      "type": "object",
+      "required": ["uri", "cid", "did", "creator", "displayName", "indexedAt"],
+      "properties": {
+        "uri": { "type": "string", "format": "at-uri" },
+        "cid": { "type": "string", "format": "cid" },
+        "did": { "type": "string", "format": "did" },
+        "creator": { "type": "ref", "ref": "app.bsky.actor.defs#profileView" },
+        "displayName": { "type": "string" },
+        "description": {
+          "type": "string",
+          "maxGraphemes": 300,
+          "maxLength": 3000
+        },
+        "descriptionFacets": {
+          "type": "array",
+          "items": { "type": "ref", "ref": "app.bsky.richtext.facet" }
+        },
+        "avatar": { "type": "string" },
+        "policies": {"type": "ref", "ref": "app.bsky.label.defs#labelerPolicies"},
```

Comment:
may be worth having a `#labelerView` & `#labelerViewBasic` where the basic view doesn't include policies. Seems a little heavy duty to be sending down in every embed for instance

Reply:
Ah yes, I forgot about embeds in general. Will add that